app/services/questions_service.rb

```diff
@@ -0,0 +1,104 @@
+module QuestionsService
```

Comment:
I don't think this is a service.  These belong with the model or in the controller.  If you have service, you need specs.

Reply:
FYI @ebrett @mo-zag I am polishing the two we have now. If we need a third I can chip in.